# Created at 2020-09-23T16:52:29.047
TestSet has not finished before stream error has appeared >> initializing exit by non-null configuration: DEFAULT
java.io.EOFException
	at java.base/java.io.DataInputStream.readInt(DataInputStream.java:397)
	at org.apache.maven.surefire.booter.MasterProcessCommand.decode(MasterProcessCommand.java:115)
	at org.apache.maven.surefire.booter.CommandReader$CommandRunnable.run(CommandReader.java:391)
	at java.base/java.lang.Thread.run(Thread.java:834)


# Created at 2020-09-23T16:52:30.474
Unexpected IOException with stream: org.apache.hadoop.fs.PathIsDirectoryException: `/home/alberto/Escritorio/outputFileTest': Is a directory

# Created at 2020-09-23T16:52:30.475
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.476
Unexpected IOException with stream: 	at org.apache.hadoop.fs.FileUtil.checkDest(FileUtil.java:518)

# Created at 2020-09-23T16:52:30.477
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.477
Unexpected IOException with stream: 	at org.apache.hadoop.fs.FileUtil.checkDest(FileUtil.java:520)

# Created at 2020-09-23T16:52:30.478
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.478
Unexpected IOException with stream: 	at org.apache.hadoop.fs.FileUtil.copy(FileUtil.java:397)

# Created at 2020-09-23T16:52:30.479
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.479
Unexpected IOException with stream: 	at org.apache.hadoop.fs.FileUtil.copy(FileUtil.java:387)

# Created at 2020-09-23T16:52:30.480
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.480
Unexpected IOException with stream: 	at org.apache.hadoop.fs.FileUtil.copy(FileUtil.java:337)

# Created at 2020-09-23T16:52:30.481
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.482
Unexpected IOException with stream: 	at org.apache.hadoop.fs.FileSystem.copyToLocalFile(FileSystem.java:2434)

# Created at 2020-09-23T16:52:30.483
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.484
Unexpected IOException with stream: 	at org.apache.hadoop.fs.FileSystem.copyToLocalFile(FileSystem.java:2403)

# Created at 2020-09-23T16:52:30.485
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.486
Unexpected IOException with stream: 	at org.apache.hadoop.fs.FileSystem.copyToLocalFile(FileSystem.java:2379)

# Created at 2020-09-23T16:52:30.486
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.487
Unexpected IOException with stream: 	at Hadoop.HDFSLogic.copyToLocal(HDFSLogic.java:356)

# Created at 2020-09-23T16:52:30.488
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.488
Unexpected IOException with stream: 	at Hadoop.HadoopLogic.downloadResult(HadoopLogic.java:230)

# Created at 2020-09-23T16:52:30.489
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.489
Unexpected IOException with stream: 	at EnviromentClass.DatabaseAnalizerLinkClass.executeAnalizerWorkFlow(DatabaseAnalizerLinkClass.java:56)

# Created at 2020-09-23T16:52:30.490
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.490
Unexpected IOException with stream: 	at EnviromentClass.DatabaseAnalizerLinkClass.executeWorkFlowDatabaseAnalizer(DatabaseAnalizerLinkClass.java:50)

# Created at 2020-09-23T16:52:30.490
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.491
Unexpected IOException with stream: 	at com.TFG_BIG_DATA.maven.TFG_MAVEN_2.LogicEnviroment.executeWOrkFlow(LogicEnviroment.java:174)

# Created at 2020-09-23T16:52:30.491
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.492
Unexpected IOException with stream: 	at com.TFG_BIG_DATA.maven.TFG_MAVEN_2.LogicEnviroment.<init>(LogicEnviroment.java:163)

# Created at 2020-09-23T16:52:30.492
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.492
Unexpected IOException with stream: 	at com.TFG_BIG_DATA.maven.TFG_MAVEN_2.ModelViewController.<init>(ModelViewController.java:32)

# Created at 2020-09-23T16:52:30.493
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.493
Unexpected IOException with stream: 	at EnviromentTest.BigDataWorkFlowTest.test(BigDataWorkFlowTest.java:27)

# Created at 2020-09-23T16:52:30.494
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.494
Unexpected IOException with stream: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)

# Created at 2020-09-23T16:52:30.494
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.495
Unexpected IOException with stream: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)

# Created at 2020-09-23T16:52:30.495
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.495
Unexpected IOException with stream: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)

# Created at 2020-09-23T16:52:30.496
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.496
Unexpected IOException with stream: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)

# Created at 2020-09-23T16:52:30.496
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.497
Unexpected IOException with stream: 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)

# Created at 2020-09-23T16:52:30.497
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.497
Unexpected IOException with stream: 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)

# Created at 2020-09-23T16:52:30.498
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.498
Unexpected IOException with stream: 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)

# Created at 2020-09-23T16:52:30.498
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.499
Unexpected IOException with stream: 	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)

# Created at 2020-09-23T16:52:30.499
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.499
Unexpected IOException with stream: 	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)

# Created at 2020-09-23T16:52:30.500
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.500
Unexpected IOException with stream: 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)

# Created at 2020-09-23T16:52:30.500
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.500
Unexpected IOException with stream: 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)

# Created at 2020-09-23T16:52:30.501
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.501
Unexpected IOException with stream: 	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)

# Created at 2020-09-23T16:52:30.501
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.501
Unexpected IOException with stream: 	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)

# Created at 2020-09-23T16:52:30.502
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.502
Unexpected IOException with stream: 	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)

# Created at 2020-09-23T16:52:30.502
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.503
Unexpected IOException with stream: 	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)

# Created at 2020-09-23T16:52:30.503
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.503
Unexpected IOException with stream: 	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)

# Created at 2020-09-23T16:52:30.503
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.503
Unexpected IOException with stream: 	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)

# Created at 2020-09-23T16:52:30.504
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.504
Unexpected IOException with stream: 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)

# Created at 2020-09-23T16:52:30.504
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.504
Unexpected IOException with stream: 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)

# Created at 2020-09-23T16:52:30.505
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.505
Unexpected IOException with stream: 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)

# Created at 2020-09-23T16:52:30.505
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.505
Unexpected IOException with stream: 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)

# Created at 2020-09-23T16:52:30.506
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.506
Unexpected IOException with stream: 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)

# Created at 2020-09-23T16:52:30.506
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.506
Unexpected IOException with stream: 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)

# Created at 2020-09-23T16:52:30.507
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.507
Unexpected IOException with stream: 	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)

# Created at 2020-09-23T16:52:30.507
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.507
Unexpected IOException with stream: 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)

# Created at 2020-09-23T16:52:30.507
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.510
Unexpected IOException with stream: path de las alida de hadoop  /home/alberto/Escritorio/outputFileTest   nombre del fichero /part-r-00000


# Created at 2020-09-23T16:52:30.512
Unexpected IOException with stream: 


# Created at 2020-09-23T16:52:30.512
Unexpected IOException with stream: JAJAJA true


# Created at 2020-09-23T16:52:30.513
Unexpected IOException: 6,1,EnviromentTest.BigDataWorkFlowTest,test(EnviromentTest.BigDataWorkFlowTest),null,null,null


# Created at 2020-09-23T16:52:30.514
Unexpected IOException: 2,1,org.apache.maven.surefire.junit4.JUnit4Provider,EnviromentTest.BigDataWorkFlowTest,null,null,null


